boards.txt

```diff
@@ -335,6 +335,49 @@ lolin32.menu.UploadSpeed.512000.upload.speed=512000
 
 ##############################################################
 
+lolin32-lite.name=WEMOS LOLIN32-Lite
+
+lolin32-lite.upload.tool=esptool
+lolin32-lite.upload.maximum_size=1310720
+lolin32-lite.upload.maximum_data_size=294912
+lolin32-lite.upload.wait_for_upload_port=true
+
+lolin32-lite.serial.disableDTR=true
+lolin32-lite.serial.disableRTS=true
+
+lolin32-lite.build.mcu=esp32
+lolin32-lite.build.core=esp32
+lolin32-lite.build.variant=lolin32-lite-lite
```

Comment:
This is wrong, the variant folder is `lolin32-lite` in this patch. Therefore the correct value would be:
`lolin32-lite.build.variant=lolin32-lite`

Reply:
Correct in #1076